LICENSE

```diff
@@ -341,10 +384,15 @@ http://www.lovelysystems.com
 Copyright Lovely Systems GmbH
 License: Apache 2.0 (http://www.apache.org/licenses)
 
+FIXME: can we specifically identify what this refers to? if not, let's remove
+
 
 ================================================================================
 
 Apache XBean
 http://geronimo.apache.org/xbean/
 Copyright Apache Software Foundation
 License: Apache License v2 (http://www.apache.org/licenses)
+
+FIXME: I cannot find this source code in our Git repository. am I correct in
+thinking this is no longer used or is a build time inclusion?
```

Comment:
@mxm

Reply:
This is still used, see /app/build.gradle

Reply:
@mxm but is this included in the source or do we fetch it for the the build? for the purposes of LICENSE, we only care about the former. not the latter

Reply:
It's a binary included in the build process.